e2core/server/handlers.go

```diff
@@ -72,140 +76,7 @@ func (s *Server) executePluginByNameHandler() echo.HandlerFunc {
 
 		responseData := seq.Request().State[mod.FQMN]
 
-		return c.Blob(http.StatusOK, "application/octet-stream", responseData)
-	}
-}
-
-func (s *Server) executePluginByRefHandler(l zerolog.Logger) echo.HandlerFunc {
-	ll := l.With().Str("handler", "executePluginByRefHandler").Logger()
-
-	return func(c echo.Context) error {
-		ref := c.Param("ref")
-
-		mod := s.syncer.GetModuleByRef(ref)
-		if mod == nil {
-			return echo.NewHTTPError(http.StatusNotFound, "module not found").SetInternal(fmt.Errorf("no module by ref %s", ref))
-		}
-
-		ll.Debug().Str("fqmn", mod.FQMN).Msg("found module by ref")
-
-		req, err := request.FromEchoContext(c)
-		if err != nil {
-			return echo.NewHTTPError(http.StatusInternalServerError).SetInternal(err)
-		}
-
-		err = req.UseSuborbitalHeaders(c)
-		if err != nil {
-			return err
-		}
-
-		steps := []tenant.WorkflowStep{{FQMN: mod.FQMN}}
-
-		// a sequence executes the handler's steps and manages its state.
-		seq, err := sequence.New(steps, req)
-		if err != nil {
-			return echo.NewHTTPError(http.StatusInternalServerError, "failed to handle request").SetInternal(err)
-		}
-
-		if err := s.dispatcher.Execute(seq); err != nil {
-			return echo.NewHTTPError(http.StatusInternalServerError, "failed to execute plugin").SetInternal(err)
-		}
-
-		// handle any response headers that were set by the Runnables.
-		if req.RespHeaders != nil {
-			for head, val := range req.RespHeaders {
-				// need to directly assign because .Add and .Set will filter out non-standard
-				// header names, which ours are.
-				if c.Response().Header()[head] == nil {
-					c.Response().Header()[head] = make([]string, 0)
-				}
-
-				c.Response().Header()[head] = append(c.Response().Header()[head], val)
-			}
-		}
-
-		responseData := seq.Request().State[mod.FQMN]
-
-		return c.Blob(http.StatusOK, "application/octet-stream", responseData)
-	}
-}
-
-func (s *Server) executeWorkflowHandler() echo.HandlerFunc {
```

Comment:
same as above, the code path no longer exists. Was only called when admin was NOT enabled